pymc3/backends/ndarray.py

```diff
@@ -72,17 +72,21 @@ def load_trace(directory, model=None):
     pm.Multitrace that was saved in the directory
     """
     straces = []
-    for directory in glob.glob(os.path.join(directory, '*')):
-        if os.path.isdir(directory):
-            straces.append(SerializeNDArray(directory).load(model))
+    for subdir in glob.glob(os.path.join(directory, '*')):
+        if os.path.isdir(subdir):
+            straces.append(SerializeNDArray(subdir).load(model))
+    if straces == []:
```

Comment:
should be `if len(straces) == 0` (or `if not straces`), since testing list equality is dangerous

Reply:
Thanks. Everything should be in place now.  ISTR getting a pylint complaint about checking length as a way to check for an empty list, so I went for `if not straces`.

Reply:
Ugh, I hate that warning!